### DISCUSSION TITLE
Best way to send Error Log event to Webhook

### DISCUSSION THREAD

**Discussion by gwilymgj** [spatie/laravel-discord-alerts#42] [Q&A]
Hi - I'd like to use laravel-discord-alerts to also send Error logs to the Web hook - what is the best way to achieve that? Could I define a Log Channel in logging.php like below and make it work?

PS: I'm using this library to send alerts to a Matrix web hook because they use the Discord format. Very cool.

<img width="601" alt="image" src="https://github.com/spatie/laravel-discord-alerts/assets/1432243/369bcd8a-585e-443a-8d31-1bcaba28a458">

**Comment by lpoirothattermann** [comment]
Hello, @Nielsvanpach @freekmurze, do you know if there is a workaround to make it work?
Thank you for your work! :)
